dataactcore/scripts/pullFPDSData.py

```diff
@@ -1340,12 +1340,26 @@ def get_delete_data(contract_type, now, sess, last_run, start_date=None, end_dat
     i = 0
     logger.info('Starting delete feed: %sCONTRACT_TYPE:"%s"', delete_url + params, contract_type.upper())
     while True:
-        resp = requests.get(delete_url + params + 'CONTRACT_TYPE:"' + contract_type.upper() + '"&start=' + str(i),
-                            timeout=60)
-        resp_data = xmltodict.parse(resp.text, process_namespaces=True,
-                                    namespaces={'http://www.fpdsng.com/FPDS': None,
-                                                'http://www.w3.org/2005/Atom': None,
-                                                'https://www.fpds.gov/FPDS': None})
+        exception_retries = -1
```

Comment:
Are we getting timeouts in the delete feed too?

Reply:
Yeah, I got them while testing these changes.